python/res/job_queue/queue.py

```diff
@@ -260,20 +260,16 @@ def getJobStatus(self, job_number):
         int_status = self._get_job_status(job_number)
         return JobStatusType( int_status )
 
-    def is_running(self):
+    def is_active(self):
```

Comment:
have you tested whether `is_alive` was enough, but basically it's fine

Reply:
This will not be enough, as it will not include jobs ready to be started.

Reply:
ok